### PR TITLE
Home learn more content

### PR DESCRIPTION
**Pull request by gonzalo1330**

### CODE REVIEW

**Comment by gonzalo1330** [comment]
One question I have is: is it possible to align text within the grid to be centered but from top-down not centered in the sides? If there is can you point me towards the right direction or an API that shows those attributes I looked on the material UI but couldn't find it but I might have missed it because of the wording possible. Aside from that then the render looks similar to the mock design. This portion will go below PR #89

File: src/components/LearnMore.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+export default function LearnMore() {
+  return (
+    <Container style={{ textAlign: 'left', paddingTop: '50px' }}>
+      <Grid container spacing={2}>
+        <Grid item>
+          <img width="500" src={logopic} className="profilePic" alt="" />
+        </Grid>
+        <Grid item xs={12} sm={6} container>
+          <Grid item xs container direction="column" spacing={2}>
```
Comment (edwingl10) [inline src/components/LearnMore.jsx:17]: For vertical centering try `style={{ alignItems:'center' }}`
I believe it should work for the first container but if not maybe try it on the the one below.

File: src/App.css [hunk 2]
```diff
@@ -1,5 +1,6 @@
 .App {
   text-align: center;
```
Comment (josueetcom) [inline src/App.css:2]: Please remove this so that you can left align below

File: src/App.css [hunk 3]
```diff
@@ -1,5 +1,6 @@
...
   width: 100vw;
-  min-height: 100vh;
+  min-height: 90vh;
+  background-color: white;
```
Comment (josueetcom) [inline src/App.css:5]: Please revert these changes. The last line conflicts with the theme's background color

**Review by josueetcom — COMMENTED** [review]
> One question I have is: is it possible to align text within the grid to be centered but from top-down not centered in the sides? If there is can you point me towards the right direction or an API that shows those attributes I looked on the material UI but couldn't find it but I might have missed it because of the wording possible. Aside from that then the render looks similar to the mock design. This portion will go below PR #89

I'm not sure exactly what you mean but you can always set `textAlign` on parent elements and that'll affect their children. Additionaly you can try exploring the [interactive Grid example](https://material-ui.com/components/grid/#interactive) to see if there's a combination of `direction`, `justify` and `alignItems` you can use.

File: src/components/LearnMore.jsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { Container } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+import logopic from '../img/blank.png';
+
+export default function LearnMore() {
+  return (
+    <Container style={{ textAlign: 'left', paddingTop: '50px' }}>
```
Comment (josueetcom) [inline src/components/LearnMore.jsx:11]: `textAlign` shouldn't need to be set. Try inspecting this element in the browser and figuring what's changing the text alignment and fix that instead.

`paddingTop` is not necessary here either

**Comment by josueetcom** [comment]
> One question I have is: is it possible to align text within the grid to be centered but from top-down not centered in the sides? If there is can you point me towards the right direction or an API that shows those attributes I looked on the material UI but couldn't find it but I might have missed it because of the wording possible. Aside from that then the render looks similar to the mock design. This portion will go below PR #89

Ah, I think I know what you mean now. This is the reason you added the `paddingTop` field huh? Well, based on the [interactive Grid example](https://material-ui.com/components/grid/#interactive) it looks like what you want is to set `justify` to `flex-start` and `alignItems` to `center`. But that assumes that you add each element as a Grid item in a container. So your code would look something like

```jsx
<Grid container>
  <Grid item container xs={12} sm={6}>  // You'll need to style justify and alignItems
    <Grid item>
       <Typography>...</Typography>
    </Grid>
    // repeat for each element
  </Grid>
  <Grid item xs={12} sm={6}>
  </Grid>
</Grid>
```

File: src/components/LoginDialog.jsx [hunk 5]
```diff
@@ -10,14 +10,18 @@ import PropTypes from 'prop-types';
 
 const uiConfig = {
   signInFlow: 'popup',
+  signInSuccessWithAuthResult() {
+    return true;
+  },
+  signInSuccessUrl: '/get-started',
```
Comment (josueetcom) [inline src/components/LoginDialog.jsx:16]: This file's changes do not seem related to this PR

File: src/components/LearnMore.jsx [hunk 6]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { Container } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+import logopic from '../img/blank.png';
+
+export default function LearnMore() {
```
Comment (josueetcom) [inline src/components/LearnMore.jsx:9]: The structure of this component is very similar to the other ones that are going to be on the home page. Something to think about for a later PR is how you might combine the two into a more generic `HomeSection` component that take in things like the title, image, body, and direction.

File: src/App.css (hunk 3 above)
Comment (gonzalo1330) [reply src/App.css:5]: Ok removed them from there

File: src/App.css (hunk 2 above)
Comment (gonzalo1330) [reply src/App.css:2]: ok removed so the only thing on the CSS file should be : 
```css
.App {
  width: 100vw;
}
```

File: src/components/LearnMore.jsx (hunk 4 above)
Comment (gonzalo1330) [reply src/components/LearnMore.jsx:11]: Ok got it figured out how to resolve this
& also paddingTop is removed but there might need to be space above once the component `ScholarshipMadeSimple` is above

File: src/components/LoginDialog.jsx (hunk 5 above)
Comment (gonzalo1330) [reply src/components/LoginDialog.jsx:16]: what should I do? remove them or keep them

File: src/components/LearnMore.jsx (hunk 1 above)
Comment (gonzalo1330) [reply src/components/LearnMore.jsx:17]: Right that does work for horizontal but I was messing around with the web inspector and discovered that align-self: 'center' does the centering for vertical position. Thanks for the help btw

**Comment by gonzalo1330** [comment]
I believe the changed should allow this PR to merge with main its looking exactly as the mock design and the img issue was fixed along with the setting of styles inline. If any changes are needed ill get them done once they are requested

File: src/components/LearnMore.jsx (hunk 6 above)
Comment (gonzalo1330) [reply src/components/LearnMore.jsx:21]: ok ill consider how to get this done

File: src/components/LoginDialog.jsx (hunk 5 above)
Comment (josueetcom) [reply src/components/LoginDialog.jsx:16]: Remove them from this PR as they're not related.
